app/components/Button/index.tsx

```diff
@@ -33,6 +33,9 @@ type Props = {
 
   /** Make it look like a link */
   flat?: boolean;
+
+  /** Ghost button styling */
+  ghost?: boolean;
```

Comment:
Cool style, buuut weird name choice? Why not just `outline`?

Reply:
`ghost` is the industry name for it - I didn't make it up.

Reply:
Oh what. Alright, still kinda weird :shrug: